addons/website/static/tests/tours/snippet_visibility_option.js

```diff
@@ -0,0 +1,113 @@
+            if (!isBlockInvisible || !isColumnInvisible) {
+                console.error("Visibility issue detected with the elements.");
+            }
+        }
+    },
+    ...wTourUtils.clickOnSave(),
+    ...wTourUtils.clickOnEditAndWaitEditMode(),
+    {
+        content: "Click on the banner snippet in the list of invisible elements.",
+        trigger: "li > .o_we_invisible_entry",
+        run: "click"
+    },
+    {
+        content: "Verify that the popup is visible and the column is still invisible.",
+        trigger: "li > .o_we_invisible_entry",
+        run: () => {
+            const isPopupVisible = document.querySelector(".o_we_invisible_root_parent i").classList.contains("fa-eye");
+            const isColumnInvisible = document.querySelector("li li .o_we_invisible_entry i").classList.contains("fa-eye-slash");
+            if (!isPopupVisible || !isColumnInvisible) {
+                console.error("Visibility issue detected with the elements.");
+            }
+        }
+    },
+    ...wTourUtils.clickOnSave(),
+    ...wTourUtils.clickOnEditAndWaitEditMode(),
+    {
+        content: "Click on the column snippet in the list of invisible elements.",
+        trigger: "li li .o_we_invisible_entry",
+        run: "click"
+    },
+    {
+        content: "Verify that both the popup and the banner are now visible.",
+        trigger: ".o_we_invisible_root_parent",
+        run: () => {
+            const isPopupVisible = document.querySelector(".o_we_invisible_root_parent i").classList.contains("fa-eye");
+            const isBlockVisible = document.querySelector("li .o_we_invisible_entry i").classList.contains("fa-eye");
+            if (!isPopupVisible || !isBlockVisible) {
+                console.error("Visibility issue detected with the elements.");
+            }
+        }
+    },
+    {
+        content: "Click on the popup snippet to hide its descendant elements.",
+        trigger: ".o_we_invisible_root_parent",
+        run: () => {
+            setTimeout(() => {
+                document.querySelector(".o_we_invisible_root_parent").click();
+            }, 1000);
+        }
+    },
+    {
+        content: "Make sure the parent snippet is invisible.",
+        trigger: ".o_we_invisible_root_parent i.fa-eye-slash",
+    },
+    {
+        content: "Verify that both the banner and column snippets are marked as invisible.",
+        trigger: ".o_we_invisible_root_parent",
+        run: () => {
+            const isBlockInvisible = document.querySelector("li .o_we_invisible_entry i").classList.contains("fa-eye-slash");
+            const isColumnInvisible = document.querySelector("li li .o_we_invisible_entry i").classList.contains("fa-eye-slash");
+            if (!isColumnInvisible || !isBlockInvisible) {
+                console.error("Visibility issue detected with the elements.");
+            }
+        }
+    },
+]);
```

Comment:
Could you add a test following the steps I outlined in my initial comment on the PR once you've addressed the issue?